src/libs/ReportUtils.js

```diff
@@ -1693,6 +1693,17 @@ function canEditReportAction(reportAction) {
     );
 }
 
+/**
+ * Gets all distance transactions on an IOU report
+ *
+ * @param {string|null} iouReportID
```

Comment:
We prefer uppercase `String` in JSDocs. Also it is required to be a string based on the prop type. We should update this JSDoc too https://github.com/Expensify/App/blob/b0b2b5751157866712877cc6224f526e505b729a/src/libs/ReportUtils.js#L1725

Reply:
Oh and I don't think we actually need this function anymore. It is now unused.